crates/nu-command/src/platform/ansi/ansi_.rs

```diff
@@ -1,3 +1,4 @@
+use dialoguer::console::strip_ansi_codes;
```

Comment:
We should probably not use dialoguer here and instead `nu_utils::strip_ansi_likely()` or `strip_ansi_string_likely()`

Reply:
Sorry - was just looking at one of the commits. I think you may have already done this.